packages/react-core/src/components/Form/examples/Form.md

```diff
@@ -67,7 +67,7 @@ class SimpleForm extends React.Component {
             <Popover
               headerContent={
                 <div>
-                  The <a href="https://schema.org/name">name</a> of a <a href="https://schema.org/Person">Person</a>
+                  The{' '}<a href="https://schema.org/name" target="_blank">name</a>{' '}of a{' '}<a href="https://schema.org/Person" target="_blank">Person</a>
```

Comment:
seems our documentation is stripping out these whitespaces? I don't see that on codesandbox @redallen

Reply:
I see this in our docs:
```jsx
headerContent={
  <div>
    The{' '}<a href="https://schema.org/name" target="_blank">name</a>{' '}of a{' '}<a href="https://schema.org/Person" target="_blank">Person</a>
  </div>
}
```
which matches codesandbox:
```jsx
headerContent={
  <div>
    The{' '}<a href="https://schema.org/name" target="_blank">name</a>{' '}of a{' '}<a href="https://schema.org/Person" target="_blank">Person</a>
  </div>
}
```

What's the discrepancy @jschuler ?

Reply:
This PR is adding `{' '}` to have spaces show in the headerContent, because without it the space is ignored in the PF docs. But even without them the space should be visible (which it is if you look at codesandbox for example)

Reply:
Ah you're right. I'll see if I can fix this upstream.